setup.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='RecoUtils',
+      version='2019.02',
+      description='Reco Utils',
+      author='Microsoft',
+      packages=['reco_utils','reco_utils.common','reco_utils.dataset','reco_utils.evaluation','reco_utils.recommender'],
```

Comment:
not including azureml?

Reply:
this should only be the list of packages within this project, and not dependencies.

Reply:
This is just to create a package (distribution) from github repo, right? The ask is to make sure our purpose is just to distribute a package of the repo onto PyPi for easy downloading, or to ease the installation as well (with the help of `distutils`. If there is just a zipped package distribution for downloading, after that, a user is expected to run `python setup install` for a particular environment probably an option parameter should be specified.

We may want to have advice from @gjwgit and @simonzhaoms as they have a nice setup script in mlhub which may be helpful in our case. See [here](https://github.com/mlhubber/mlhub/blob/master/setup.py).